contracts/cw20-stake-external-rewards/src/contract.rs

```diff
@@ -463,6 +544,32 @@ pub fn execute_update_manager(
         ))
 }
 
+pub fn execute_update_pause_status(
+    deps: DepsMut,
+    env: Env,
+    info: MessageInfo,
+    is_paused: bool,
```

Comment:
naming nit: can we name this var `pause` instead of `is_paused` since its requesting an action? (